ui/temboardui/templates/flask/dashboard.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+
+{% block title %}{{ super() }}/ Dashboard{% endblock %}
+
+{% block content %}
+<div id="app">
+  <dashboard
+    :config="{{ config }}"
+    :instance="{
+      'hostname': '{{ instance.hostname }}',
+      'agentAddress': '{{ instance.agent_address }}',
+      'agentPort': '{{ instance.agent_port }}',
+      'pgPort': '{{ instance.pg_port }}',
+      'plugins': {{ instance.plugins | map(attribute='plugin_name') | list }}
+    }"
+    :discover="{{ instance.discover }}"
+    :jdata-history="{{ history }}"
+    :initial-data="{{ dashboard }}"
+  />
+</div>
+
+<!-- <script src="/js/Chart.js"></script>
+-->
```

Comment:
While we're at it, can we remove this commented code? I'm sure it was added for debug. Let's remove it.